docs/UserGuide.md

```diff
@@ -298,8 +298,7 @@ by adding your first Delivery
 
 </box>
 
-2. After registering for an account, you should be greeted by some sample Customers
-   that have been generated by HomeBoss.
+2. After registering for an account, you should be greeted by a empty homepage, let's populate it with some data.
```

Comment:
Should we perhaps complete the statement here i.e.,
"After registering for an account, you will be greeted by HomeBoss's home page. It's empty at the moment, so let's populate it with some data."